i2c/README.md

```diff
@@ -4,39 +4,35 @@ To get the embedded driver for your Sensirion Sensor working there are now
 only a few steps missing. These steps are needed since our driver is made
 in a way that it is possible to run it on different hardware platforms.
 So we need to adjust two files according to your platform.
-
-
-
 The first one is `sensirion_i2c_hal.c`.
-In this file it is implemented how to send your requests to the Sensor.
-How this is done depends on your hardware platform therefore we can't
+In this file it is implemented how to send your requests to the sensor.
+How this is done depends on your hardware platform. Therefore we can't
 provide just one version of this code. The default version of this file
-used is an empty one, where you could implement it yourself. Luckily
-for most common platforms we provide already implemented versions in
+used is an empty one, where you could implement it yourself. For
```

Comment:
I don't understand this sentence: `The default version of this file used is an empty one, where you could implement it yourself.` So it's an empty file we provide? That means the user needs to implement it not `could`, right?

Reply:
Is this better? The default version of this file is an empty template, where you can implement it yourself.